project/src/main/java/com/google/sps/data/User.java

```diff
@@ -183,8 +172,9 @@ public String getEmail() {
     return this.eventsHosting;
   }
 
-  public Set<Long> getEventsHostingIds() {
-    Set<Long> eventIds = new HashSet<>();
+  /** Returns a Set of eventHosting Id fields */
+  public Set<String> getEventsHostingIds() {
+    Set<String> eventIds = new HashSet<>();
     for (Event event : this.eventsHosting) {
       eventIds.add(event.getId());
     }
```

Comment:
Try this?

eventHosting.stream().map(Event::getId).collect(Collectors.toSet());

Reply:
Neat, thanks!